packages/terra-action-footer/tests/jest/CenteredActionFooter.test.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import CenteredActionFooter from '../../src/CenteredActionFooter';
+
+describe('CenteredActionFooter', () => {
+  // Snapshot Tests
+  describe('Snapshot Tests', () => {
+    it('should render an empty footer', () => {
+      const component = <CenteredActionFooter />;
+      expect(shallow(component)).toMatchSnapshot();
+    });
+
+    it('should render a footer with an anchor', () => {
+      const component = (
+        <CenteredActionFooter
+          center={<a href="/">Test</a>}
+        />
+      );
+      expect(shallow(component)).toMatchSnapshot();
+    });
+
+    it('should render a footer with two buttons', () => {
+      const component = (
+        <CenteredActionFooter
+          center={
+            <div>
+              <button>Button 1</button>
+              <button>Button 2</button>
+            </div>
+          }
+        />
+      );
+      expect(shallow(component)).toMatchSnapshot();
+    });
+  });
+
+  // Prop Tests
```

Comment:
Can snapshots be used for these prop tests instead? It looks like it's testing the exact same things the snapshot would test.

Reply:
For sure. Is that the best practice that we should follow moving forward?

Reply:
That's my recommendation. The snapshot is doing the same stuff here as checking prop values, and it takes care of all the demeter logic it takes to ensure props are the values they should be.